src/popup/App.vue

```diff
@@ -19,12 +19,16 @@
     >
       <Header v-if="showHeader" />
 
-      <transition :name="$route.meta.asModal ? 'pop-transition' : 'page-transition'">
-        <RouterView
-          :class="{ 'show-header': showHeader }"
-          class="main"
-        />
-      </transition>
+      <div class="app-container">
+        <transition :name="$route.meta.asModal ? 'pop-transition' : 'page-transition'">
```

Comment:
Transition component should be put immediately above the element that changes in order to apply the transition. The structure has changed and there is no page transition at all.